apps.yml

```diff
@@ -240,7 +240,7 @@ apps:
   - application:
       name: "Egencia (FR)"
       op: auth0
-      url: "http://www.egencia.fr/pub/agent.dll?qscr=vain&amp;vain=mozilla"
+      url: " http://mozilla.egencia.fr"
```

Comment:
Should there be a space between the quote and the http?

Reply:
Good catch. I recreated the PR #152